soccer/SimFieldView.cpp

```diff
@@ -68,9 +69,17 @@ void SimFieldView::mousePressEvent(QMouseEvent* me) {
 }
 
 void SimFieldView::mouseMoveEvent(QMouseEvent* me) {
+    _posLabel->move(QPoint(me->pos().x() - 45, me->pos().y() + 17));
```

Comment:
Yeah sounds good.  I think you can still move this to `FieldView`, you'll just have to reimplement the `mouseMoveEvent()` in that class the same way it's implemented here.  The only caveat is that in order for both `SimFieldView` and it's parent class (`FieldView`) to have the mouseMovedEvent() method called, you'll need to call it here in this class.

I'd recommend moving this code block into `FieldView::mouseMoveEvent()`, then putting `FieldView::mouseMoveEvent(me)` here.